.github/workflows/check-pr-title.yml

```diff
@@ -1,3 +1,5 @@
+---
+
```

Comment:
is this really needed?

Reply:
the yaml-linter told me so... (I could of course add a config file for the yaml linter, and tell him that I don't care about the three dashes, but that would be too much effort).

Reply:
That feels like tooling working against you...